### PR TITLE
Start/stop rgb/depth order issue

### PR DESCRIPTION
**Pull request by WeilongYE**
Prevents to start/stop color streaming when streaming depth.

### CODE REVIEW

File: src/astra_driver.cpp [hunk 1]
```diff
@@ -484,8 +497,8 @@ void AstraDriver::imageConnectCb()
     if (!color_started)
     {
       if (enable_streaming_) {
-        ROS_INFO("Starting color stream.");
-        device_->startColorStream();
+        ROS_INFO("Starting color stream. Fake start!");
+        //device_->startColorStream();
```
Comment (danielhailuA) [inline src/astra_driver.cpp:501]: What is the purpose of this commented code ? enable_streaming_ does nothing.

File: src/astra_driver.cpp [hunk 2]
```diff
@@ -497,8 +510,8 @@ void AstraDriver::imageConnectCb()
 
     if (color_started)
     {
-      ROS_INFO("Stopping color stream.");
-      device_->stopColorStream();
+      ROS_INFO("Stopping color stream. Fake Stop!");
+      //device_->stopColorStream();
```
Comment (danielhailuA) [inline src/astra_driver.cpp:514]: commented code

File: src/astra_driver.cpp (hunk 1 above)
Comment (WeilongYE) [reply src/astra_driver.cpp:501]: Do nothing.

**Comment by WeilongYE** [comment]
Declined, this PR is include in https://github.com/avidbots/ros_astra_camera/pull/9

**Pull request closed without merging**